Tasks/KubernetesV1/task.loc.json

```diff
@@ -154,7 +154,7 @@
       "name": "useConfigurationFile",
       "type": "boolean",
       "label": "ms-resource:loc.input.label.useConfigurationFile",
-      "defaultValue": "false",
+      "defaultValue": "true",
```

Comment:
Setting this default value to true matching the current state, to avoid back-compat issues.